src/client/components/footer.js

```diff
@@ -31,50 +31,92 @@ function Footer(props) {
 
 	return (
 		<footer className="footer">
-			<Grid fluid>
-				<Row>
-					<Col xs={4}>
-						<small>{'Tested with '}
-							<a
-								href="https://www.browserstack.com/"
-								rel="noopener noreferrer"
-								target="_blank"
-							>
-								<img
-									alt="BrowserStack Logo"
-									height="25"
-									src="/images/BrowserStack.png"
-								/>
-							</a>
-						</small>
+			<Grid fluid className="padding-top-1 padding-bottom-1">
```

Comment:
I've had better results adding these `padding-top-1 padding-bottom-1` to this Grid's two Row children